__tests__/app.test.js

```diff
@@ -90,6 +90,7 @@ describe("GET /api/articles/:article_id", () => {
         expect(article.article_img_url).toBe(
           "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"
         );
+        expect(article.comment_count).toBe("11");
```

Comment:
Nice, but ideally when adding a new "feature" to an existing endpoint, I would recommend creating a new test for it instead of amending previously written tests